gluster-recycle-pv.sh

```diff
@@ -106,54 +106,25 @@ done
 for pv_name in "${pvs[@]}"; do
 
   # Get size of existing PV
-  OIFS="$IFS"
-  IFS=';'
-  pv_info=($(oc get pv "$pv_name" -o template --template='{{.spec.capacity.storage}};{{.spec.glusterfs.path}};{{.spec.glusterfs.endpoints}}' 2>/dev/null))
-  pv_size=${pv_info[0]}
-  pv_path=${pv_info[1]}
-  pv_ep=${pv_info[2]}
-  IFS="$OIFS"
+  pv_path=$(oc get pv "$pv_name" -o template --template='{{.spec.glusterfs.path}}' 2>/dev/null)
```

Comment:
I know this was in the code before, but there's no reason to hide errors. Drop the stderr redirection.